Strategic/Game Clock.cpp

```diff
@@ -45,6 +45,9 @@ BOOLEAN fClockMouseRegionCreated = FALSE;
 
 BOOLEAN fTimeCompressHasOccured = FALSE;
 
+// run time compression till next hour?
+BOOLEAN fStopTimeCompressionNextHour = FALSE;

```

Comment:
Since this is new code, you can just use `bool`. Also no need for the "f" prefix.